doc/release-notes/0.33/0.33.md

```diff
@@ -0,0 +1,115 @@
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.33.0 release notes
+
+These release notes support the [Eclipse OpenJ9 0.33.0 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.33.0/plan).
+
+## Supported environments
+
+OpenJ9 release 0.33.0 supports OpenJDK 8, 11, 17, 18.
+
+All releases are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests provided by Adoptium.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.33.0. Further information about these changes can be found in the [user documentation](https://eclipse-openj9.github.io/openj9-docs/version0.33/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr>
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/11164">#11164</a></td>
+<td valign="top">Early access release for Apple Silicon macOS</td>
+<td valign="top">Java 11 and later / Apple Silicon macOS</td>
+<td valign="top">Build for Apple Silicon (AArch64) macOS is available as an early access release.  It is stable enough for evaluation but not suitable for production yet.</td>
+</tr>
+
+</tbody>
+</table>
+
+## Known issues
+
+The v0.33.0 release contains the following known issues and limitations:
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1">
+<thead align="left">
+<tr>
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+<th valign="bottom">Workaround</th>
+</tr>
+</thead>
+
+<tbody>
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/15011">#15011</a></td>
+<td valign="top">The default stack size is modified for the main thread</td>
+<td valign="top">All</td>
+<td valign="top">The main thread stack size was 1MB in releases prior to 0.32. In the 0.32 release and later it's modified to a smaller platform
+dependent value, the same value as the <tt>-Xmso</tt> setting. The 0.33 release increases the default <tt>-Xmso</tt> stack size on x64 platforms, but
+Java 17 and later also requires more stack space to run. These changes may result in a <tt>java.lang.StackOverflowError: operating system stack overflow</tt></td>
```

Comment:
Suggest
* hyphenating "platform-dependent"
* requires -> require (in "Java 17 and later also require ...")
* the final sentence should end with a period

Reply:
Updated